Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -254,9 +252,10 @@ public static void ResetLaunchCount()
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="SystemInformation"/> class.
```

Comment:
The line right below this is mentioning that it "Initializes static members...", which is not the case anymore.